app/report/report.controller.js

```diff
@@ -3,13 +3,17 @@
 (function() {
   angular.module('reportController', ['selectedFilesService']).
 
+  controller('ReportSelectionController', ['$scope', '$routeSegment', function($scope, $routeSegment) {
+    $scope.$routeSegment = $routeSegment;
+  }]).
+
   controller('ReportController', ['$scope', 'FileList', 'SelectedFiles', function($scope, FileList, SelectedFiles) {
     $scope.records = SelectedFiles;
 
-    $scope.sort_property = 'format';
-    $scope.reverse = false;
-    $scope.sort_fn = function(record) {
-      var sort_prop = $scope.sort_property;
+    $scope.format_sort_property = 'format';
```

Comment:
Not harming anything, but do we still need these if the tables are in separate tabs?